src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_NamingRule.cs

```diff
@@ -45,18 +45,18 @@ internal static partial class EditorConfigNamingStyleParser
             return false;
         }
 
-        private static DiagnosticSeverity ParseEnforcementLevel(string ruleSeverity)
+        private static ReportDiagnostic ParseEnforcementLevel(string ruleSeverity)
         {
             switch (ruleSeverity)
             {
                 case EditorConfigSeverityStrings.None:
```

Comment:
>EditorConfigSeverityStrings.None: [](start = 21, length = 33)

None still maps to hidden and not suppressed?

Reply:
➡️ You are reviewing commit 26de379ca11ef3cc2d6e6d037d9440776355300a, which is a behavior preserving change. A later commit, 8305bae9672ca55b796fb2aafee28e8ad25756ea, implements the new design.